### PR TITLE
Use python and runtime version

### PR DESCRIPTION
**Pull request by monochromegane**
We need to specify new parameters.

When we create a job (runtimeVersion, pythonVersion).
- https://cloud.google.com/ml-engine/docs/training-jobs

When we create a version of model (framework, runtimeVersion, pythonVersion). 
- https://cloud.google.com/ml-engine/docs/deploying-models

### CODE REVIEW

File: starchart/main.py [hunk 1]
```diff
@@ -25,7 +27,10 @@ def main():
 
     expose_parser = subparsers.add_parser('expose',  help='Expose model')
     expose_parser.add_argument('-p', '--project-id',  help="Project ID. Or environment value named GCP_PROJECT_ID.")
-    expose_parser.add_argument('-m', '--model-name', required=True, help="Model name.")
+    expose_parser.add_argument('-m', '--model-name',      required=True, help="Model name.")
+    expose_parser.add_argument('-R', '--runtime-version', help="Runtime version for job.")
```
Comment (zaimy) [inline starchart/main.py]: for model?

File: starchart/main.py [hunk 2]
```diff
@@ -25,7 +27,10 @@ def main():
...
+    expose_parser.add_argument('-P', '--python-version',  help="Python version for job.")
```
Comment (zaimy) [inline starchart/main.py]: also, I think this argument is for model.

**Review by zaimy — COMMENTED** [review]
Thank you very much for your work!
I have some questions about help with command line arguments.

**Review by zaimy — APPROVED** [review]
LGTM!!!

**Comment by monochromegane** [comment]
Thanks! Merry X'mas!

**Comment by zaimy** [comment]
🎄

**Pull request merged**